src/components/TodoForm.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useContext } from "react";
+import { TodoContext } from "./TodoContext";
+import moment from "moment"; 
+import "./TodoForm.css"
+
+export const TodoForm = () => {
+  const [todo, setTodo] = useState("");
+  const { addTodo } = useContext(TodoContext);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    
+    if (!todo) return;
+
+    addTodo({
+      id: Date.now(),
```

Comment:
We also took the same approach to create a "unique" ID; love that you thought the same 👏 

-Linda&Etna